includes/Utilities/TraverseHelpers.php

```diff
@@ -7,21 +7,37 @@
 
 namespace WPGraphQL\ContentBlocks\Utilities;
 
+use WPGraphQL\ContentBlocks\Data\ContentBlocksResolver;
+
 /**
  * Class TraverseHelpers
  *
  * Provides utility functions to traverse and manipulate blocks.
+ *
+ * @deprecated @todo Blocks should be manipulated directly inside ContentBlocksResolver::handle_do_block()
  */
 final class TraverseHelpers {
 	/**
 	 * Traverse blocks and apply a callback with optional depth limit.
 	 *
+	 * @deprecated @todo Blocks should be manipulated directly inside ContentBlocksResolver::handle_do_block()
+	 *
 	 * @param array    &$blocks The blocks to traverse.
 	 * @param callable $callback The callback function to apply to each block.
 	 * @param int      $depth The current depth of traversal.
 	 * @param int      $max_depth The maximum depth to traverse.
 	 */
 	public static function traverse_blocks( &$blocks, $callback, $depth = 0, $max_depth = PHP_INT_MAX ): void {
+		_deprecated_function(
+			__METHOD__,
+			'@todo',
+			sprintf(
+			// translators: %s: class name
+				esc_html__( 'Manipulate blocks directly inside %s::handle_do_block', 'wp-graphql-content-blocks' ),
```

Comment:
missing sprintf?

Reply:
In addition to the one on L34?

Reply:
nah we're good. i think it was a presentation thing in the Github diff. I see it now.